ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -42,21 +42,30 @@
     "system": "system",
 }
 
+SUPPORTED_OPERATIONS = ["llm",
+                        "chat",
+                        "chain",
+                        "embedding"
+                        ]
+
 
 class LangChainIntegration(BaseLLMIntegration):
     _integration_name = "langchain"
 
     def llmobs_set_tags(
-        self,
-        operation: str,  # oneof "llm","chat","chain"
-        span: Span,
-        inputs: Any,
-        response: Any = None,
-        error: bool = False,
+            self,
+            operation: str,  # oneof "llm","chat","chain","embedding"
+            span: Span,
+            inputs: Any,
+            response: Any = None,
+            error: bool = False,
     ) -> None:
         """Sets meta tags and metrics for span events to be sent to LLMObs."""
         if not self.llmobs_enabled:
             return
+        if operation not in SUPPORTED_OPERATIONS:
+            log.warning("Unsupported operation : %s" % operation)
```

Comment:
Thanks for adding the unsupported case check! If the operation is not supported, can we return immediately instead?

Reply:
Yes sure ! I was thinking that we still want to see the spans with general tagging if it was an operation that wasn't dealt with from our side yet.

Reply:
This should technically never get run, but still great error handling. If it's an operation we don't support but somehow end up in `llmobs_set_tags(...)`, we may actually result in errors trying to format/extract data that isn't there outside of the chat/chain/llm/embedding scenario, so I think we should just return instead.